iml-agent/src/action_plugins/kernel_module.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::{agent_error::ImlAgentError, cmd::cmd_output};
+
+pub async fn loaded(module: String) -> Result<bool, ImlAgentError> {
+    let output = cmd_output("lsmod", vec![]).await?;
+
+    if output.status.success() {
+        let stdout = String::from_utf8_lossy(&output.stdout);
+        let mut modules = stdout.lines().into_iter().filter_map(|m| {
+            let mut fields = m.split(' ').filter(|s| *s != "");
+            let (name, _, refcount) = (fields.next(), fields.next(), fields.next());
+            let refcount = refcount.map(|x| x.parse::<u32>());
+            if let (Some(name), Some(Ok(refcount))) = (name, refcount) {
+                Some((name, refcount))
+            } else {
+                None
+            }
+        });
+
+        let module = modules.find(|(m, refcount)| *m == module && *refcount > 0);
```

Comment:
I think refcount is the number of other modules requiring this one, and thus can be 0, but the module is loaded.

Reply:
I got info from [RedHat](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/4/html/Reference_Guide/s2-proc-modules.html)

> The third column lists how many instances of the module are currently loaded. A value of zero represents an unloaded module.

But now I see [this](https://stackoverflow.com/questions/47645859/meaning-of-modules-instances-in-proc-modules).

So I guess even RedHat misdocuments :man_shrugging: 

Fixing now.